packages/block-editor/src/components/block-card/index.js

```diff
@@ -2,22 +2,48 @@
  * WordPress dependencies
  */
 import deprecated from '@wordpress/deprecated';
-
+import { Button } from '@wordpress/components';
+import { chevronLeft, chevronRight } from '@wordpress/icons';
+import { __, isRTL } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
 import BlockIcon from '../block-icon';
 
-function BlockCard( { title, icon, description, blockType } ) {
+function BlockCard( {
+	title,
+	icon,
+	description,
+	blockType,
+	parentBlockClientId,
+	handleBackButton,
+} ) {
 	if ( blockType ) {
 		deprecated( '`blockType` property in `BlockCard component`', {
 			since: '5.7',
 			alternative: '`title, icon and description` properties',
 		} );
 		( { title, icon, description } = blockType );
 	}
+
+	const isOffCanvasNavigationEditorEnabled =
+		window?.__experimentalEnableOffCanvasNavigationEditor === true;
+
 	return (
 		<div className="block-editor-block-card">
+			{ isOffCanvasNavigationEditorEnabled && parentBlockClientId && (
```

Comment:
This should also just work for the navigation block

Reply:
Should it? Why not do it for all child blocks?

Reply:
I'd like to query how folks think we can do that _without_ tightly coupling the block editor to a specific block.

Also I think it's probably great for _all_ blocks 😄